docs/diga_api.md

```diff
@@ -0,0 +1,216 @@
+        <bas:Name_Kommunikationsstelle>DIGA-BITMARCK</bas:Name_Kommunikationsstelle>
+        <bas:Endpunkt_Kommunikationsstelle>diga.bitmarck-daten.de</bas:Endpunkt_Kommunikationsstelle>
+        <bas:Versandart>1</bas:Versandart>
+        <bas:Postalische_Zusaetze>BMW BKK</bas:Postalische_Zusaetze>
+        <bas:Strasse_Postfach>Mengkofener Str.</bas:Strasse_Postfach>
+        <bas:Hausnummer_Postfachnummer>6</bas:Hausnummer_Postfachnummer>
+        <bas:PLZ>84130</bas:PLZ>
+        <bas:Ort>Dingolfing</bas:Ort>
+        <bas:Kontaktdaten_Technisch_Telefon>0800-24862725</bas:Kontaktdaten_Technisch_Telefon>
+        <bas:Kontaktdaten_Technisch_EMail>servicedesk@bitmarck.de</bas:Kontaktdaten_Technisch_EMail>
+</n1:Krankenkasseninformation>
+```
+
+The file can be updated by insurances which requires keeping the mapping file used for the client up-to-date. The field `Kostentraegerkuerzel` is what can be used to find insurance information based on the prescription code. The insurance information lists a field `Endpunkt_Kommunikationsstelle`. This is the base url for the endpoint for a specific insurance. There is no central diga endpoint for verifying and reimbursing DiGA apps and handling might differ among insurances. Some apis might also not support billing as of now. Invoices in these cases must be sent by email or post.
+
+Although insurances use their own apis, they do follow an openapi specification for the request which can be found [here](https://www.gkv-datenaustausch.de/media/dokumente/leistungserbringer_1/digitale_gesundheitsanwendungen/technische_anlagen_aktuell_7/digaSP_1_0_05.yaml). This [documents](https://github.com/alex-therapeutics/diga-api-client/blob/main/ENDPOINT_STATUS.md) lists which apis are currently working with the client.
+
+## Request format
+
+According to the openapi specification the request contains 4 different parameters which are sent with `ContentType: multipart/form-data`:
+
+- `iksender` - Institutionskennzeichen (IK) of the DiGA manufacturer (an id required for payments)
+- `ikempfaenger` - IK of the insurance which can be found in the mapping file (`Kostentraegerkennung` - [section 5.2.2](https://www.gkv-datenaustausch.de/media/dokumente/leistungserbringer_1/digitale_gesundheitsanwendungen/technische_anlagen_aktuell_7/DiGA_Anlage_1_Technische_Anlage_zur_RL_V1.1_20210225.pdf))
+- `verfahren` - a code which discriminates requests for code verification and reimbursement
+- `nutzdaten` - additional information for the request, including the prescription code
+  - the `nutzdaten` are described later in more detail as this is the tricky bit of the request
+
+### Institutionskennzeichen - IK numbers
+
+Detailed information about the IK numbers can be found [here](https://www.gkv-datenaustausch.de/media/dokumente/leistungserbringer_1/GR_IK_2020-06-01.pdf) (German only).
+
+> Das IK ist ein eindeutiges Merkmal für die Abrechnung medizinischer und rehabilitativer
```

Comment:
Do we need this quote?

Reply:
Shortened it a bit. Found it would be nice to have quotes s.t. you can easily search in the linked documents.